salt/reviewer-suggestions/init.sls

```diff
@@ -91,6 +92,19 @@ reviewer-suggestions-app-cfg:
         - require:
             - reviewer-suggestions-repository
 
+reviewer-suggestions-newrelic-cfg:
+    file.managed:
+        - user: {{ pillar.elife.deploy_user.username }}
+        - name: /srv/reviewer-suggestions/client/.inject-html/newrelic.html
+        - source: 
+            - salt://reviewer-suggestions/config/srv-reviewer-suggestions-client-inject-html-newrelic-{{ pillar.elife.env }}.html
+            - salt://reviewer-suggestions/config/srv-reviewer-suggestions-client-inject-html-newrelic-default.html
+        - template: jinja
```

Comment:
I think you can remove `template: jinja` as the files do not need to be rendered